docs/MAINTAINING.md

```diff
@@ -82,3 +82,84 @@ npm dist-tag add vtex@2.91.0 latest
 3. Make sure that we aren't enforcing the deprecated version (see [forcing a new version](##forcing-a-new-version))
 
 That's it!
+
+### Changing the release message
+
+Release notes are updated every month. Initially we manually updated the message on toolbelt and released
+a new version for every new release notes, but this flow had problems like:
+
+- The user had to update the CLI to see the new release message.
+- We had to go for the trouble of releasing a new version just for updating the release notes message.
+
+Because of this the app [vtex.toolbelt-config-server](https://github.com/vtex/toolbelt-config-server) was extended
+to provide messages via a REST API. When toolbelt has to show the release notes message it does a request to
+`vtex.toolbelt-config-server` and, with that, we can just update the message via the REST API and all users will
+start to see the updated message. The messages provided by the toolbelt-config-server are documented [here](https://github.com/vtex/toolbelt-config-server#messages).
+
+Now, here's how to update the release notes message:
+
+1. You'll have to install the [toolbelt-config-cli](https://github.com/vtex/toolbelt-config-cli):
+
+```
+yarn global add @vtex/toolbelt-config-cli
+```
+
+2. Login to `vtex` account.
```

Comment:
What do you think about adding the command? Just to be 100% clear since the other steps have the command example
```suggestion
2. Login to `vtex` account.

`$ vtex login vtex`

```

Reply:
Nice suggestion!